pychromecast/controllers/media.py

```diff
@@ -394,9 +394,29 @@ def _fire_status_changed(self):
     # pylint: disable=too-many-arguments
     def play_media(self, url, content_type, title=None, thumb=None,
                    current_time=0, autoplay=True,
-                   stream_type=STREAM_TYPE_BUFFERED):
-        """ Plays media on the Chromecast. Start default media receiver if not
-            already started. """
+                   stream_type=STREAM_TYPE_BUFFERED,
+                   metadata=None):
+        """
+        Plays media on the Chromecast. Start default media receiver if not
+        already started.
+
+        Parameters:
+        url: str - url of the media.
```

Comment:
Great stuff.

Here's a small tip for documenting parameters in Python, if you use the restructuredtext syntax it will be picked up by the Sphinx doc generator and it is also supported by IDE's (such as PyCharm).
http://www.sphinx-doc.org/en/stable/domains.html#info-field-lists

In this case it would be:
```
:param url: url of the media
:type url: str
```